Dockerfile

```diff
@@ -1,14 +1,5 @@
 FROM nvidia/cuda:11.8.0-base-ubuntu20.04
 
-ARG NO_PROXY

```

Comment:
let's remove this from the readme if we are disabling this here (https://github.com/Sanofi-Public/spatialone-pipeline/blob/master/docs_md/cardhovernet_tutorial.md#3-set-up-the-environment)

Reply:
Done.